index.html

```diff
@@ -1,296 +1,34 @@
-          clip-rule="evenodd"
-        />
-      </svg>
-    </li>
-    <li>
-      <ul class="languages-elements">
-        <li>
-          <span class="lang-logo">
-            <img
-              class="frontend-logo js-logo"
-              src="images/js.png"
-              alt="JavaScript Logo"
-            />
-          </span>
-          <p>JavaScript</p>
-        </li>
-        <li>
-          <span>
-            <img
-              class="frontend-logo"
-              src="images/html.image.png"
-              alt="HTML Logo"
-            />
-          </span>
-          <p>HTML</p>
-        </li>
-        <li>
-          <span>
-            <img
-              class="frontend-logo"
-              src="images/css.png"
-              alt="CSS Logo"
-            />
-          </span>
-          <p>CSS</p>
-        </li>
-      </ul>
-    </li>
-    <li class="skillset-title border">
-      <h3 class="skillset-heading">Frameworks</h3>
-      <span class="sign"
-        ><svg
-          xmlns="http://www.w3.org/2000/svg"
-          viewBox="0 0 24 24"
-          fill="currentColor"
-          class="down-sign"
-        >
-          <path
-            fill-rule="evenodd"
-            d="M16.28 11.47a.75.75 0 010 1.06l-7.5 7.5a.75.75 0 01-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 011.06-1.06l7.5 7.5z"
-            clip-rule="evenodd"
-          />
-        </svg>
-      </span>
-    </li>
-    <li class="skillset-title border">
-      <h3 class="skillset-heading">Skills</h3>
-      <span class="sign"
-        ><svg
-          xmlns="http://www.w3.org/2000/svg"
-          viewBox="0 0 24 24"
-          fill="currentColor"
-          class="down-sign"
-        >
-          <path
-            fill-rule="evenodd"
-            d="M16.28 11.47a.75.75 0 010 1.06l-7.5 7.5a.75.75 0 01-1.06-1.06L14.69 12 7.72 5.03a.75.75 0 011.06-1.06l7.5 7.5z"
-            clip-rule="evenodd"
-          />
-        </svg>
-      </span>
-    </li>
-  </ul>
-</section>
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="styles.css" />
+    <title> Tribute Page</title>
+</head>
+<body>
+    <!--
+       <div id="main">
+        <div id="title"><h1>Dr Norman Borlaug</h1></div>
+        <Figure id="img-div">
+            <img id="image" src="https://cdn.freecodecamp.org/testable-projects-fcc/images/tribute-page-main-image.jpg" alt="image">
+            <div id="img-caption"><p>Dr. Norman Borlaug, third from the left, trains biologists in Mexico on how to increase wheat yields - part of his life-long war on hunger.</p></div>
+        </Figure>
+        <div id="tribute-info">
+          <a href="#" id="tribute-link" target="_blank">The man who saved a billion lives</a>
+        </div>
+    </div>
+    -->
```

Comment:
Good job so far
![image](https://user-images.githubusercontent.com/116277115/236897482-4b66ee68-fe2e-4bbf-bea3-3128a9bc0c5f.png)
 though there are some small changes to adjust looks like the code is commented out, meaning it won't be displayed on the page. It would be better to remove the commented-out code to keep the project clean and organized.